__tests__/fixtures/workflow/workflow.yml

```diff
@@ -24,6 +24,12 @@ on:
           multiline description
         type: number
         required: true
+
+    secrets:
```

Comment:
Secrets are optional, I think it is better to add a dedicated workflow to check the secret parsing. This ensure workflows without secrets are also parsed correctly.

Reply:
Good point, updated